doc/source/deployment/setup-deployer.rst

```diff
@@ -0,0 +1,179 @@
+.. _setupdeployer:
+
+Set Up Deployer
+=================
+
+.. blockdiag::
+
+   blockdiag {
+     default_fontsize = 11;
+     deployer [label="Setup deployer"]
+     ses_integration [label="SES Integration"]
+     configure [label="Configure\nCloud"]
+     setup_caasp_workers [label="Setup CaaS Platform\nworker nodes"]
+     patch_upstream [label="Apply patches\nfrom upstream\n(for developers)"]
+     build_images [label="Build Docker images\n(for developers)"]
+     deploy_airship [label="Deploy Airship"]
+     deploy_openstack [label="Deploy OpenStack"]
+
+     group {
+       deployer
+       color="red"
+     }
+
+     deployer -> ses_integration;
+     ses_integration -> configure;
+     configure -> setup_caasp_workers;
+
+     group {
+       color = "#EEEEEE"
+       label = "Cloud Deployment"
+       setup_caasp_workers -> patch_upstream;
+       patch_upstream -> build_images;
+       build_images -> deploy_airship [folded];
+       setup_caasp_workers -> deploy_airship;
+       deploy_airship -> deploy_openstack;
+     }
+   }
+
+
+Base Software
+-------------
+
+The following software must be installed on your `Deployer`:
+
+  * ansible>=2.7.0
+  * gcc
+  * git
+  * jq
+  * python3-netaddr
+  * python-virtualenv
+
+Create SUSE Containerized OpenStack Workspace
+---------------------------------------------
+
+All the deployment artifacts are stored in a :term:`workspace`. The workspace
+by default is a directory located in the user's home directory on the Deployer.
+Create a directory in your home directory that ends in -workspace. Then export
+SOCOK8S_ENVNAME=<directory name prefix> to set your workspace. To change your
+workspace parent directory, export `SOCOK8S_WORKSPACE_BASEDIR` with the base
+directory where your workspace is located.
+
+.. code-block:: console
+
+  mkdir ~/socok8s-workspace
+  export SOCOK8S_ENVNAME=socok8s
+
+
+Cloning repository
```

Comment:
are we gonna use git only or package only or both? There is currently a package available so this should change I guess. Depends on if we want to provide a unstable version (git) or a know, tested, packaged version.